From bebb70054694731392acdd2245b98bdaa6be9f2d Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Mon, 19 Jan 2009 12:33:29 +0530
Subject: [PATCH 185/193] Temperory workarond for ISP-CCDC hang issue

We have known issue for ISP-CCDC, where CCDC goes into infinite
wait loop due to the race condition between ISP interrupt and
buffer management under ISR.
Will be addressed as part of stabilization period.
---
 drivers/media/video/isp/isp.c |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/isp/isp.c b/drivers/media/video/isp/isp.c
index 87d85dd..c8dc167 100644
--- a/drivers/media/video/isp/isp.c
+++ b/drivers/media/video/isp/isp.c
@@ -1502,7 +1502,10 @@ void isp_vbq_done(unsigned long status, isp_vbq_callback_ptr arg1, void *arg2)
 		spin_lock(&isp_obj.isp_temp_buf_lock);
 		if (ispmodule_obj.isp_temp_state == ISP_BUF_INIT) {
 			spin_unlock(&isp_obj.isp_temp_buf_lock);
-			ispccdc_enable(0);
+			/*
+			 * Temoerory workaroung for ISP-CCDC hang issue
+			 */
+//			ispccdc_enable(0);
 			return;
 		}
 		spin_unlock(&isp_obj.isp_temp_buf_lock);
-- 
1.5.6

